connector/connect/common/src/main/java/org/apache/spark/sql/api/java/UDF11.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.api.java;
+
+import org.apache.spark.annotation.Stable;
+
+import java.io.Serializable;
+
+/**
+ * A Spark SQL UDF that has 11 arguments.
+ */
+@Stable
+public interface UDF11<T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, T11, R> extends Serializable {
```

Comment:
Are UDF1~UDF22 new definitions added instead of moving from the `sql/core` module to the `common` module or `sql/api` module?

Reply:
also cc @amaliujia, Is it better to keep only one of these class definitions?

Reply:
yes thanks for the moving..